tmt/steps/__init__.py

```diff
@@ -1682,10 +1688,10 @@ def plugins(cls, step: Step) -> List['Login']:
         return [Login(logger=step._logger.descend(), step=step, order=phase)
                 for phase in cls.phases(step)]
 
-    def go(self) -> None:
+    def go(self, always=True) -> None:
```

Comment:
Other `go()` methods just gained the `force` parameter, isn't it expressing pretty much the same idea as `always`, i.e. "don't check whether you should run, run because I told you so". Would it then make sense to use `force` here as well?

Reply:
Good point! Renamed in 93c1eec1.